website/src/css/custom.css

```diff
@@ -31,3 +31,10 @@ html[data-theme='dark'] {
     --ifm-font-size-base: 17px;
   }
 }
+
+.docusaurus-highlight-code-line {
```

Comment:
Can it also make sense to add this CSS code to the template? So that the line highlighting will work initially, and it will be easier to change for end-user (no need to copy-pasting styles from docs).

https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-init/templates/classic/src/css/custom.css

Reply:
Good idea

Reply:
Good idea. My only concern is that users who don't use this but aren't aware of this line will be shipping redundant CSS.

Reply:
Well.. yes, on the other hand, we always have code in the bundle, which may not be used :man_shrugging:

Reply:
If you're referring to the other Infima CSS styles, yes they are there for now. But I plan to implement some form of CSS tree shaking in future (but for Infima only).

Reply:
I'm talking about JS code for this feature, for example.
It seems to me not so much maybe redundant CSS will be delivered to the user. After all, this is a documentation tool, and docs often use line highlighting, it seems to me that we need to proceed from this fact.

Reply:
I think this is ok to ship it on init. Few lines of css code is < 1kb. Shipping stuff such as material-ui component is not ok 😉